packages/api/src/modules/tags.module.ts

```diff
@@ -0,0 +1,14 @@
+import { Module } from "@nestjs/common";
+import { TypeOrmModule } from "@nestjs/typeorm";
+import { TransactionTag } from "@cryptify/common/src/domain/entities/TransactionTag";
+import { TagsService } from "../services/tags.service";
+import { TagsController } from "../controllers/tags.controller";
+import { UsersModule } from "./users.module";
+
+@Module({
+    imports: [TypeOrmModule.forFeature([TransactionTag]), UsersModule],
```

Comment:
I don't think you are using the UsersModule exports anywhere here

```suggestion
    imports: [TypeOrmModule.forFeature([TransactionTag])],
```